packages/openjphjs/src/HTJ2KDecoder.hpp

```diff
@@ -38,7 +38,7 @@ class HTJ2KDecoder
   {
     // Use the following for debugging to ensure updated version info
     // Update the XX to check that reload has completed
-    // OJPH_INFO(0x00010002, "vXX HTJ2K Decoder");
+    OJPH_INFO(0x00010002, "v06 HTJ2K Decoder");
```

Comment:
Temporary while we figure out why the code fails to decode

Reply:
as talked, this makes it bigger but gives stack trace etc